basics/basic-types.md

```diff
@@ -0,0 +1,124 @@
+# Grundläggande typer
+
+D erbjuder ett antal grundläggande typer som alltid har samma
+storlek **oavsett** vilken plattform. Det enda undantaget
+är typen `real` som erbjuder den högsta möjliga precision för
+flyttal. Det finns ingen skillnad mellan storleken på ett heltal
```

Comment:
This sounds wrong in my opinion.
Maybe something like: `Det finns inga skillnader för heltals storlekarna oavsett om programmet är kompilerat för ett 32-bitar eller ett 64-bitars system.`